lib/modules/platform/gerrit/index.ts

```diff
@@ -0,0 +1,543 @@
+  const change = await client.getChange(prConfig.number);
+  if (change.subject !== prConfig.prTitle) {
+    await updatePullRequestTitle(
+      prConfig.number,
+      change.change_id,
+      prConfig.prTitle
+    );
+  }
+  if (prConfig.prBody) {
+    await updatePullRequestBody(prConfig.number, prConfig.prBody);
+  }
+  if (prConfig.platformOptions?.autoApprove) {
+    await client.approveChange(prConfig.number);
+  }
+  if (prConfig.state && prConfig.state === 'closed') {
+    await client.abandonChange(prConfig.number);
+  }
+}
+
+export async function createPr(prConfig: CreatePRConfig): Promise<Pr | null> {
+  logger.debug(
+    `createPr(${prConfig.sourceBranch}, ${prConfig.prTitle}, ${
+      prConfig.labels?.toString() ?? ''
+    })`
+  );
+  const pr = (
+    await findOwnPr(
+      {
+        branchName: prConfig.sourceBranch,
+        targetBranch: prConfig.targetBranch,
+        state: 'open',
+      },
+      true
+    )
+  ).pop();
+  if (pr) {
+    //Workaround for "Known Problems.1"
+    if (pr.subject !== prConfig.prTitle) {
+      await updatePullRequestTitle(pr._number, pr.change_id, prConfig.prTitle);
+    }
+    await updatePullRequestBody(pr._number, prConfig.prBody);
+    if (prConfig.platformOptions?.autoApprove) {
+      await client.approveChange(pr._number);
+    }
+    return getPr(pr._number);
+  } else {
+    throw new Error(
+      `the change should be created automatically from previous push to refs/for/${prConfig.sourceBranch}`
+    );
+  }
+}
+
+async function updatePullRequestTitle(
+  number: number,
+  gerritChangeID: string,
+  prTitle: string
+): Promise<void> {
+  try {
+    await client.setCommitMessage(
+      number,
+      `${prTitle}\n\nChange-Id: ${gerritChangeID}\n`
+    );
+  } catch (err) {
+    logger.error(
+      { err },
+      `Can't set pull-request-title ${prTitle} as commit-msg for change ${gerritChangeID}/${number}`
+    );
+  }
+}
+
+async function updatePullRequestBody(
+  changeId: number,
+  prBody: string
+): Promise<void> {
+  const prBodyExists = await checkForExistingMessage(
+    changeId,
+    prBody,
+    TAG_PULL_REQUEST_BODY
+  );
+  if (!prBodyExists) {
+    await client.addMessage(changeId, prBody, TAG_PULL_REQUEST_BODY);
+  }
+}
+
+async function checkForExistingMessage(
+  changeId: number,
+  newMessage: string,
+  msgType: string | null
+): Promise<boolean> {
+  const newMsg = newMessage.trim(); //the last \n was removed from gerrit after the comment was added...
+  const messages = await client.getMessages(changeId);
+  return (
+    messages.find(
+      (existingMsg) =>
+        (msgType === null || msgType === existingMsg.tag) &&
+        existingMsg.message.includes(newMsg)
+    ) !== undefined
+  );
+}
+
+export async function getBranchPr(branchName: string): Promise<Pr | null> {
+  const change = (await findOwnPr({ branchName, state: 'open' })).pop();
+  if (change) {
+    return mapGerritChangeToPr(change);
+  }
+  return null;
```

Comment:
Use conditional expression like in other function, so we use same style everywhere
```suggestion
  return change ? mapGerritChangeToPr(change) : null;
```